eo-maven-plugin/src/main/java/org/eolang/maven/SafeMojo.java

```diff
@@ -272,6 +302,38 @@ public List<Tojo> select(final Predicate<Tojo> filter) {
      */
     abstract void exec() throws IOException;
 
+    /**
+     * Runs exec command with timeout if needed.
+     *
+     * @throws ExecutionException If unexpected exception happened during execution
+     * @throws TimeoutException If timeout limit reached
+     * @throws IOException If fails
+     */
+    private void execWithTimeout() throws ExecutionException, TimeoutException, IOException {
+        try {
+            if (this.timeout == null) {
```

Comment:
@volodya-lombrozo maybe, in order to simplify the code, it's better to set `timeout` to `Integer.MAX_VALUE` by default? Then, you won't need to check it for `null`

Reply:
@yegor256 done.